### DISCUSSION TITLE
Multitrack audio for a video in mp4 format and audio files

### DISCUSSION THREAD

**Discussion by superfabiuz** [videojs/video.js#8016] [General]
Hello. I have a video file in mp4 format with embedded audio in Italian. Also, I have 3 audio files with translation in 3 different languages. How can I handle language switching in javascript while keeping sync? It can be done?
It is not possible to re-encode video files by inserting multi audio tracks. I ask if there is a system to play the video with a different audio, maybe even putting buttons outside the player
Thank you very much

**Comment by StephennFernandes** [comment]
@superfabiuz hey do you have any update on this ? 
i am stuck in a similar solution.

**Comment by Stallone2K** [comment]
@superfabiuz Need Update On Your App Situation. Did You Make It Work ?
